emperor/core.py

```diff
@@ -60,12 +60,13 @@ class Emperor(object):
         identifiers in the `ordination` object.
     dimensions: int, optional
         Number of dimensions to keep from the ordination data, defaults to 5.
-    remote: bool, optional
-        Whether to load resources from a remote URL or to do it from the
-        nbextensions folder in your Jupyter installation. Note you will need to
-        import the `nbinstall` function from the util module and then call it
-        so the installation of the resources can take place. Defaults to load
-        resources from a remote location.
+    remote: bool or str, optional
```

Comment:
That's the cool thing about this change, we can just specify a relative path and it should work! 

Yoshiki Vázquez-Baeza

> On Apr 26, 2016, at 3:23 PM, Antonio Gonzalez <notifications@github.com> wrote:
> 
> In emperor/core.py:
> 
> > @@ -60,12 +60,13 @@ class Emperor(object):
> >          identifiers in the `ordination` object.
> >      dimensions: int, optional
> >          Number of dimensions to keep from the ordination data, defaults to 5.
> > -    remote: bool, optional
> > -        Whether to load resources from a remote URL or to do it from the
> > -        nbextensions folder in your Jupyter installation. Note you will need to
> > -        import the `nbinstall` function from the util module and then call it
> > -        so the installation of the resources can take place. Defaults to load
> > -        resources from a remote location.
> > +    remote: bool or str, optional
> What about the case that you only want to have a stand alone plot? Like we currently have.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>